src/main/resources/mturk/worker-ui/mturk.js

```diff
@@ -31,14 +31,14 @@ function initMturk(platformName, workerServiceUrl, experiment) {
         preview: preview
     });
     //load initial bits
-    CreativeCrowd.load();
-    CreativeCrowd.onSubmitAnswer(function (viewData, submittedData) {
+    WorkerUI.load();
```

Comment:
Dude, this is 1:1 the code from you sample. Do you really want to tell me the sample probably doesnt work?